packages/appsync-modelgen-plugin/src/visitors/appsync-typescript-visitor.ts

```diff
@@ -138,11 +137,6 @@ export class AppSyncModelTypeScriptVisitor<
       }
     });
 
-    if (isTimestampFeatureFlagEnabled) {
```

Comment:
Why do we not need this feature flag handling anymore?

Reply:
That flag is deprecated now: https://docs.amplify.aws/cli/reference/feature-flags/#addTimestampFields